### PR TITLE
fix: add name to confirmation dialog on delete

### PR DESCRIPTION
**Pull request by wtrocki**
Before
```
 rhoas kafka delete
? Confirm the name of the instance you want to delete: wtrocki
```

After
```
 rhoas kafka delete
? Confirm the name of the instance you want to delete (wtrocki) wtrocki
```

### CODE REVIEW

**Comment by wtrocki** [comment]
Blocked by #982

File: pkg/localize/locales/en/cmd/kafka_delete.en.toml [hunk 1]
```diff
@@ -42,7 +42,7 @@ one = 'Skip confirmation to forcibly delete this Kafka instance'
 
 [kafka.delete.input.confirmName.message]
 description = 'Input title for Kafka name confirmation'
-one = 'Confirm the name of the instance you want to delete:'
+one = 'Confirm the name of the instance you want to delete ({{.Name}})'
```
Comment (craicoverflow) [inline pkg/localize/locales/en/cmd/kafka_delete.en.toml:45]:
```suggestion
one = 'Confirm the name of the instance you want to delete ({{.Name}}):'
```

**Review by craicoverflow — APPROVED** [review]
This is **not** a **fix**. It is a new feature, nothing is broken.

I don't think that #982 blocks this.

**Comment by craicoverflow** [comment]
Good addition 👍🏻

**Pull request merged**